CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+- removed: Unused legacy databases. The v1 routes continue to support security message registration, but won't remember any preferences.
```

Comment:
This line appears to be for a different PR

Reply:
It is, but it somehow got lost.